src/audio/module_adapter/module/volume/volume_hifi3.c

```diff
@@ -133,6 +133,55 @@ static void vol_s24_to_s24_s32(struct processing_module *mod, struct input_strea
 		out = audio_stream_wrap(sink, out);
 	}
 }
+
+/**
+ * \brief HiFi3 enabled volume passthrough from 24/32 bit to 24/32 or 32 bit.
+ * \param[in,out] dev Volume base component device.
+ * \param[in,out] sink Destination buffer.
+ * \param[in,out] source Input buffer.
+ * \param[in] frames Number of frames to process.
+ * \param[in] attenuation factor for peakmeter adjustment (unused)
+ */
+static void vol_passthrough_s24_to_s24_s32(struct processing_module *mod,
```

Comment:
Could these volume processing functions names be cleaned up to simple s16/s24/s32. The old names are from days when volume was used for format conversion. Now with same input and output forward always we could make the names simpler.

Reply:
Also  all non-module adapter mode code should be removed from volume, but not a task for this PR.